### ISSUE TITLE
Pasting formatted text into unrelated text field causes "Ghost Image" uploads

### ISSUE THREAD

**Issue by dev7ch** [pqina/filepond#493]
### Summary
First of all: Thank you very much for this great and clean js plugin!

Pasting formatted text from a Word doc into a text field creates "Ghost Images" as png, ~~even if `allowPaste: false`~~

![image](https://user-images.githubusercontent.com/15183757/81673760-68820c00-944c-11ea-9b67-20d622272f47.png)

I am a bit unsure where to address this issue? Do you or anyone have an idea?

### How to reproduce

- having text input or text area on the same page as  the uploader
- copy some text from a word file
- paste it into a text field or area 
- uploads should appear as in the img above

### Expected behaviour

No interaction between paste Text and uploader

### Additional information

- Pasting without formatting by using `cmd + option + shift + V`  all works fine
- I am using the Uploader Component twice as different refs in a kind of multi step form (not a real `<form>`)

| Environment  | Version
|------------- | -----------
| OS           | MacOS latest
| Device       | 16" MBP 
| Browser      | Chrome latest (almost)

**Comment by rikschennink** [comment]
As this is related to FilePond core I've moved the issue there.

FilePond indeed should not catch these "copy paste" actions. So this feature needs to be improved upon. Internally checking if the target is a text input or text area will probably suffice.

In the mean time you can disable this behaviour by setting `allowPaste` to `false`

**Comment by dev7ch** [comment]
Thanks for the instant reply.

Yes i can confirm `allowPaste` is solving it for now

```
// in vue js
<FilePond
              :ref="'pond' + n"
              :allowPaste="false"
              :server="server"
              :instant-upload="false"
              // ...
/>

```

**Comment by josuemontano** [comment]
@rikschennink is anyone working on this bug? I could open a pull request if not.

**Comment by rikschennink** [comment]
> @rikschennink is anyone working on this bug? I could open a pull request if not.

Feel free to submit a PR

**Comment by rikschennink** [comment]
Should be fixed in 4.19
